public/app/plugins/datasource/loki/querybuilder/LokiQueryModeller.ts

```diff
@@ -18,12 +23,65 @@ export class LokiQueryModeller extends LokiAndPromQueryModellerBase {
     ]);
   }
 
-  renderLabels(labels: QueryBuilderLabelFilter[]) {
```

Comment:
Do we have coverage for these methods in LokiQueryModeller.test.ts?

Reply:
Yes, we have 30+ tests on `renderQuery` which uses all - `renderLabels`, `renderOperations`, `renderBinaryQueries`.

Reply:
Also we are not removing this method, it was just updated and you can find it bellow.